README.md

```diff
@@ -70,13 +70,18 @@ We follow the [IAI Python Style Guide](https://github.com/iai-group/styleguide/t
 If you use the PKG API, please cite the following paper:
 
 ```bibtex
-@misc{Bernard:2024:arXiv,
- title        = {PKG API: A Tool for Personal Knowledge Graph Management},
- author       = {Nolwenn Bernard and Ivica Kostric and Weronika Łajewska and Krisztian Balog and Petra Galuščáková and Vinay Setty and Martin G. Skjæveland},
- year         = 2024,
- eprint       = {2402.07540},
- archiveprefix = {arXiv},
- primaryclass = {cs.HC}
+@inproceedings{Bernard:2024:WWW,
```

Comment:
Nit: Please remove the fields isbn, publisher, address, and url to follow standard format.

Reply:
Done.